packages/bot-api/src/MessageHandler.test.node.ts

```diff
@@ -24,7 +24,7 @@ import UUID from 'uuidjs';
 import {ClientType} from '@wireapp/api-client/src/client';
 import {TextMessage} from '@wireapp/core/src/main/conversation/message/OtrMessage';
 import {Connection} from '@wireapp/api-client/src/connection';
-import {MessageBuilder} from '@wireapp/core/src/main/conversation/message/MessageBuilder';
+import {MessageBuilder} from '@wireapp/core';
```

Comment:
Since we now expose the `index.ts` of the core, we can access the message builder directly :)